jekyll/_cci2/docker-layer-caching.md

```diff
@@ -7,9 +7,13 @@ categories: [optimization]
 order: 70
 ---
 
-{% include beta-premium-feature.html feature='Docker Layer Caching' %}
+This document offers an overview of Docker Layer Caching (DLC), which can reduce Docker image build times on CircleCI. Docker Layer Caching is only available on specific plans. DLC is available on the Premium usage plan, on all server-installations. If you are on CircleCI's container-based plan and want to use DLC, please open a support ticket to request access.
```

Comment:
Probably worth just saying `only available on the Performance Plan.` Also, what is a Premium usage plan?

Reply:
Also, we no longer provide DLC for container customers, so let's make sure we don't include the last line. :smile: